bbeam/emails/Newsletter.tsx

```diff
@@ -0,0 +1,190 @@
+import { MjmlSection, MjmlColumn, MjmlImage, MjmlWrapper } from "mjml-react";
+import Layout from "./components/Layout";
+import Footer from "./components/Footer";
+import Button from "./components/Button";
+import Heading from "./components/Heading";
+import Text from "./components/Text";
+import List from "./components/List";
+import Link from "./components/Link";
+import { fontSize, colors, borderRadius, lineHeight } from "./theme";
+import assetUrl from "./util/assetUrl";
+import Spacer from "./components/Spacer";
+
+type NewsletterProps = {
+  email: string;
+};
+
+const Header = () => {
+  return (
+    <MjmlWrapper
+      fullWidth={true}
+      padding="32px 20px"
+      backgroundColor={colors.yellow}
+    >
+      <MjmlSection cssClass="gutter">
+        <MjmlColumn>
+          <MjmlImage
+            width="43px"
+            height="48px"
+            align="center"
+            src={assetUrl("/assets/logo-yellow.png")}
+            paddingBottom={16}
+          />
+          <Text align="center" fontSize={fontSize.sm}>
+            Newsletter brought to you by Bbeam
+          </Text>
+        </MjmlColumn>
+      </MjmlSection>
+    </MjmlWrapper>
+  );
+};
+
+const NewsletterSection = ({ children }: { children: React.ReactNode }) => (
+  <>
+    <MjmlSection
+      cssClass="margin-gutter gutter"
+      backgroundColor={colors.white}
+      borderRadius={borderRadius.base}
+      border={`1px solid ${colors.black}`}
```

Comment:
@elliothursh do you know if there's an easy way to make this black border go around the white edge? i.e. not touching the content. if there's not a <5 min solution, i'll just remove it... 

<img width="592" alt="Screen Shot 2022-12-05 at 3 42 31 PM" src="https://user-images.githubusercontent.com/609038/205769122-9ef8228d-cf4e-4362-8369-4e1faa9e2620.png">